app/components/project-geometries/modes/draw/feature-label-form.js

```diff
@@ -55,6 +55,23 @@ export default class FeatureLabelFormComponent extends Component {
     this.drawStateCallback();
   }
 
+  // action to hide the "Add" option in power-select-with-create if current search input matches options
+  @action
+  hideCreateOptionIfAlreadyExists(input) {
+    const options = this.get('options'); // array of district options for power-select
+    // array of districts that match substring currently being typed into search box
+    const searchOptions = options.filter(d => d.includes(input));
+    // array of all elements that match between options array and searchOptions array
+    const matchArray = options.filter(function(item) {
+      return searchOptions.includes(item);
+    });
+    // if matchArray has no elements, return true and show "Add"
+    if (matchArray.length < 1) {
+      return true;
+    }
+    return false;
+  }
```

Comment:
Why does this return something? Should it be a computed property? Maybe I don't understand `showCreateWhen` enough... but if this works, very interesting...

Reply:
I followed the way that they did it here in the docs https://github.com/cibernox/ember-power-select-with-create, where they used an action. "You can provide a callback showCreateWhen, which will be called whenever the user types into the search field. If you return true, the create option will be shown. If you return false, it won't be shown." Let me know if you still think it should be something else

Reply:
I followed the way that they did it here in the docs https://github.com/cibernox/ember-power-select-with-create, where they used an action. "You can provide a callback showCreateWhen, which will be called whenever the user types into the search field. If you return true, the create option will be shown. If you return false, it won't be shown." Let me know if you still think it should be something else